server/src/test/java/org/opensearch/index/IndexSettingsTests.java

```diff
@@ -1011,15 +955,15 @@ public void testSetRemoteTranslogBufferIntervalDefaultSetting() {
         Version createdVersion = VersionUtils.randomVersionBetween(random(), Version.V_2_0_0, Version.CURRENT);
         Settings settings = Settings.builder()
             .put(IndexMetadata.SETTING_INDEX_VERSION_CREATED.getKey(), createdVersion)
-            .put(IndexMetadata.SETTING_REMOTE_TRANSLOG_STORE_ENABLED, true)
+            .put(IndexMetadata.SETTING_REMOTE_STORE_ENABLED, true)
             .build();
         assertEquals(TimeValue.timeValueMillis(650), IndexSettings.INDEX_REMOTE_TRANSLOG_BUFFER_INTERVAL_SETTING.get(settings));
     }
 
     public void testSetRemoteTranslogBufferIntervalFailsWhenEmpty() {
         Settings indexSettings = Settings.builder()
             .put(IndexMetadata.SETTING_REPLICATION_TYPE, ReplicationType.SEGMENT)
-            .put(IndexMetadata.SETTING_REMOTE_TRANSLOG_STORE_ENABLED, false)
+            .put(IndexMetadata.SETTING_REMOTE_STORE_ENABLED, false)
```

Comment:
Is this supposed to have a `true` value for this test?

Reply:
just kept status quo here. I think the value here doesnt matter since the flow breaks for the buffer interval value parsing